src/hotspot/share/nmt/nmtTreap.hpp

```diff
@@ -409,7 +412,8 @@ class Treap {
       const int cmp_from = COMPARATOR::cmp(head->key(), from);
       const int cmp_to = COMPARATOR::cmp(head->key(), to);
       if (cmp_from >= 0 && cmp_to < 0) {
-        f(head);
+        if (!f(head))
+          return;
```

Comment:
Style: Always use braces in if statements.

Reply:
Done.